### ISSUE TITLE
Installation hangs.

### ISSUE THREAD

**Issue by antoan** [luminrobotics/ros-blessed#2]
Ubuntu 20.04
ROS Noetic

After having installed nodejs and nmp, installing ros-blessed yields the following and hangs:
```
sudo npm install -g ros-blessed                                                                                   
[..................] | fetchMetadata: sill resolveWithNewModule ros-blessed@0.2.1 checking installable status
```

**Comment by chfritz** [comment]
Hi @antoan , could you post the result of `npm --versions`?

**Comment by antoan** [comment]
npm --versions                                                                                               
{ npm: '6.14.4',
  ares: '1.15.0',
  brotli: '1.0.7',
  cldr: '36.1',
  http_parser: '2.9.3',
  icu: '66.1',
  modules: '64',
  napi: '5',
  nghttp2: '1.40.0',
  node: '10.19.0',
  openssl: '1.1.1d',
  tz: '2022a',
  unicode: '13.0',
  uv: '1.34.2',
  v8: '6.8.275.32-node.55',
  zlib: '1.2.11' }

**Comment by chfritz** [comment]
Those versions are very old! Current LTS version of node is 16. Can you install a recent version from https://github.com/nodesource/distributions and try again?

**Comment by chfritz** [comment]
PS: Looks like we may need to update our Readme.

**Comment by antoan** [comment]
I see. I reinstalled Node 16 LTS as you suggested and attempted to install blessed again.

Here is what I did. 

- Uninstalled npm & nodejs

- Installed Node.js LTS (v16.x):

curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
sudo apt-get install -y nodejs

- sudo npm install -g ros-blessed         
     
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! code 128
npm ERR! An unknown git error occurred
npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/chfritz/rosjs.git
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-06-10T20_51_25_222Z-debug-0.log


Cheers

**Comment by chfritz** [comment]
This new error means that you don't have an active github ssh key. But rather than fixing this error, please just try again now. I've just released a new version that no longer uses the github dependency. It should work now, but please let me know either way.

**Comment by antoan** [comment]
I get the same, I'm afraid.

- sudo apt update
- sudo npm install -g ros-blessed      
                                                                                       
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! code 128
npm ERR! An unknown git error occurred
npm ERR! command git --no-replace-objects ls-remote ssh://git@github.com/chfritz/rosjs.git
npm ERR! git@github.com: Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-06-10T23_26_55_496Z-debug-0.log

**Comment by chfritz** [comment]
That's bizarre. The latest version doesn't use that package anymore:
```json
{
  "name": "ros-blessed",
  "version": "0.2.2",
  "description": "A blessed ('curses') toolbox for ROS. Like rqt but for the command line.",
  "homepage": "https://github.com/luminrobotics/ros-blessed",
  "main": "index.js",
  "scripts": {
    "test": "mocha --exit *.test.js",
    "test-on-save": "while inotifywait -e close_write *.js; do npm test; done"
  },
  "bin": {
    "ros-blessed": "index.js"
  },
  "repository": {
    "type": "git",
    "url": "github:luminrobotics/ros-blessed"
  },
  "keywords": [
    "ros",
    "robotics",
    "curses"
  ],
  "author": "Christian Fritz",
  "license": "MIT",
  "dependencies": {
    "blessed": "^0.1.81",
    "lodash": "^4.17.20",
    "rosnodejs": "^3.1.0",
    "tf-rosnodejs": "^0.3.1"
  },
  "devDependencies": {
    "mocha": "^8.1.3"
  }
}
```

**Comment by antoan** [comment]
I am not familiar with node/npm. Do I need to refresh the npm's registry similar to apt update, I wonder?

**Comment by chfritz** [comment]
no, there is no such thing. Every time you run `npm install XYZ` it will look for the latest version of XYZ and install or update to it.

Maybe try forcing it to the latest by running
```
sudo npm install -g ros-blessed@0.2.2
```
